instances.go

```diff
@@ -249,3 +347,36 @@ func (c *Client) SetInstancePolicies(ctx context.Context, setDualAuth, dualAuthE
 
 	return nil
 }
+
+type portsMetadata struct {
+	CollectionType string `json:"collectionType"`
+	NumberOfPorts  int    `json:"collectionTotal"`
+}
+
+type portResponse struct {
+	Metadata portsMetadata `json:"metadata"`
+	Ports    []privatePort `json:"resources"`
```

Comment:
This may be correct, but it looks out of place. Should the json be `ports` instead of `resources`?

Reply:
resources is fine as we define our json output for all our entities as resources